openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -88,32 +88,35 @@
 - control_key: AU-3
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-156
+        When the OpenShift API audit system is enabled and configured it generates
+        audit records containing information that establishes the type of event that occurred,
+        when the event occurred, where the event occurred, the source of hte event,
+        the outcome of the event, and the idenity of the individuales or subjects associated
+        with the event.
 
 - control_key: AU-3 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-157
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
```

Comment:
What are the specific configuration actions, or links to docs.redhat.com content, that outline how to enrich the audit information?

Reply:
@shawndwells I added content that describes installing and configuring OCP cluster logging with links.  Let me know if anything needs to be addressed.